src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2661,6 +2661,507 @@ Note: `aparse()` is case-insensitive.
+            <td>yyyy-MM-dd HH:mm:ss.SSS</td>
+            <td>2023-10-18 15:27:03.123</td>
+            <td>2023-10-18T15:27:03.123Z</td>
+          </tr>
+          <tr>
+            <td>Missing datetime field</td>
+            <td>
+              Any missing time field will be replaced with 0.<br/>
+              Any missing date field will be replaced with 1.<br/>
+              If a field is present, all higher precedence fields must also be present.<br/>
+              Note that day of year and quarter of year patterns are supported.
+            </td>
+            <td>
+              yyyy-MM-dd HH:mm:ss<br/>
+              yyyy-MM-dd HH:mm<br/>
+              yyyy-MM-dd HH<br/>
+              yyyy-MM-dd<br/>
+              yyyy 'day' D<br/>
+              yyyy-MM<br/>
+              yyyy qqq<br/>
+              yyyy
+            </td>
+            <td>
+              2023-10-18 15:27:03<br/>
+              2023-10-18 15:27<br/>
+              2023-10-18 15<br/>
+              2023-10-18<br/>
+              2023 day 291<br/>
+              2023-10<br/>
+              2023 Q4<br/>
+              2023
+            </td>
+            <td>
+              2023-10-18T15:27:03.000Z<br/>
+              2023-10-18T15:27:00.000Z<br/>
+              2023-10-18T15:00:00.000Z<br/>
+              2023-10-18T00:00:00.000Z<br/>
+              2023-10-18T00:00:00.000Z<br/>
+              2023-10-01T00:00:00.000Z<br/>
+              2023-10-01T00:00:00.000Z<br/>
+              2023-01-01T00:00:00.000Z<br/>
+            </td>
+          </tr>
+          <tr>
+            <td>Time only</td>
+            <td>
+              If a time pattern is used without a date, the millisecond of the day will be calculated.  Technically, this is the unix timestamp on January 1, 1970.<br/>
+              Note: The timezone adjustment is still honored.
+            </td>
+            <td>
+              HH:mm<br/>
+              HH:mm O<br/>
+              HH:mm O
+            </td>
+            <td>
+              00:30<br/>
+              00:30 GMT-1<br/>
+              00:30 GMT+1
+            </td>
+            <td>
+              1971-01-01T00:30:00.000Z<br/>
+              1971-01-01T01:30:00.000Z<br/>
+              1969-12-31T23:30:00.000Z
+            </td>
+          </tr>
+          <tr>
+            <td>12 hour time</td>
+            <td>If a 12 hour pattern (lower case h) is used, then the am-pm-of-day pattern (a) must also be used. Note, in the datetime string AM/PM must be upper case.</td>
+            <td>
+              yyyy-MM-dd h:mm a<br/>
+              yyyy-MM-dd h:mm<br/>
+              yyyy-MM-dd h:mm a
+            </td>
+            <td>
+              2023-10-18 3:27 PM<br/>
+              2023-10-18 3:27<br/>
+              2023-10-18 3:27 pm
+            </td>
+            <td>
+              2023-10-18T15:27:00.000Z<br/>
+              Unsupported pattern<br/>
+              null (due to lowercase pm)
```

Comment:
Line breaks don't work to align the corresponding values in adjacent columns, since the longer values wrap to take up two lines and the shorter ones don't.  I think these all need to be separate nested <tr>s.

Reply:
Sounds good, I updated the table to use separate nested table rows so that the corresponding values in adjacent columns will be aligned.